### PR TITLE
fix: Fix percy glitches

### PR DESCRIPTION
**Pull request by untitaker**
Related: #12068 #12090

### CODE REVIEW

File: src/sentry/utils/samples.py [hunk 1]
```diff
@@ -202,4 +202,8 @@ def create_sample_event(project, platform=None, default=None,
 
     manager = EventManager(data)
     manager.normalize()
-    return manager.save(project.id, raw=raw)
+    event = manager.save(project.id, raw=raw)
+    # Prevent Percy screenshot from constantly changing
+    event.datetime = datetime(2017, 9, 6, 0, 0)
```
Comment (dcramer) [inline src/sentry/utils/samples.py]: is this going to cause any issues w/ the user triggered sample events given its in the past? e.g. we have an expected 90 day retention on things, and often enforce retention of search filters

Comment (untitaker) [reply src/sentry/utils/samples.py]: Good point, I will move this codeblock in tests.

I suspect this was broken before (by me) as there used to be a hardcoded timestamp as well (for reasons I don't understand)

**Review by dcramer — APPROVED** [review]
we could also explicitly pass `current_datetime` into create_sample_event, but imo this is fine

**Review by mitsuhiko — APPROVED** [review]

**Comment by untitaker** [comment]
gah, yeah that would improve perf theoretically. I don't want to wait for CI again :(

**Pull request merged**